css-two-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  width: 50%;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  line-height: 1.5;
+}
+
+.section {
+  display: flex;
+}
+
+.name-and-description {
+  flex-basis: 60%;
+  margin-top: 30px;
+}
+
+.icon {
+  flex-basis: 25%;
+  width: 100%;
```

Comment:
These will compete with one another. Choose the one that is actually the value you want, and remove the other